docs/Makefile

```diff
@@ -223,3 +223,21 @@ dummy:
 	$(SPHINXBUILD) -b dummy $(ALLSPHINXOPTS) $(BUILDDIR)/dummy
 	@echo
 	@echo "Build finished. Dummy builder generates no files."
+
+LINKSDIR       = source/links
+.PHONY: link
+link:
```

Comment:
This was for generating global links that could be shared across the documentation. I don't think I ended up using it and instead just put shared links in `shared.rst` and import that in the separate doc pages that need it.

I think this can be removed.